Deep_Learning/Alzheimers Detection/Dataset/README.md

```
@@ -0,0 +1,12 @@
+## Datasets
+The dataset used in the project is from kaggle
+link for dataset: https://www.kaggle.com/datasets/tourist55/alzheimers-dataset-4-class-of-images
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)